root/usr/src/iomemory-vsl4-4.3.7/kblock.c

```diff
@@ -54,6 +54,7 @@
 #include <linux/blk-mq.h>
 #include <linux/blkdev.h>
 #include <kblock_meta.h>
+#include <linux/bio-integrity.h>
```

Comment:
Please move this to `include/bklock-meta.h` as done here: https://github.com/RemixVSL/iomemory-vsl/blob/1034ffe936336eb2acf4fe68b5ec2c153dce4e21/root/usr/src/iomemory-vsl-3.2.16/include/kblock_meta.h#L15-L17